config/initializers/hyrax.rb

```diff
@@ -192,6 +192,8 @@
 
   ## Do not alter unless you understand how ActiveFedora handles URI/ID translation
   config.translate_id_to_uri = lambda do |id|
+    return "#{ActiveFedora.fedora.host}#{ActiveFedora.fedora.base_path}/#{::Noid::Rails.treeify(id)}" unless id.match(/^cpb-aacip/)
+    id, tail = id.split('/')
```

Comment:
For ID's that will have a "tail", are these of the form "cpb-aacip/123-456789"?

Reply:
the only two I've seen so far are
`cpb-aacip-7fdd93e156c/list_source` or `cpb-aacip-7fdd93e156c/members` but I wanted to cover the generic case so if there was something I wasn't seeing the logs we wouldn't have a sleeper on our hands